app/blueprints/fund/routes.py

```diff
@@ -72,11 +72,24 @@ def create_fund():
     """Creates a new fund"""
     form = FundForm()
 
+    params = {"form": form, "fund_id": None}
+    match request.args.get("action") or request.form.get("action"):
+        case "grants_table":
+            params["prev_nav_url"] = url_for("fund_bp.view_all_funds")
+        case "select_grant":
+            params["prev_nav_url"] = url_for("application_bp.select_fund")
+        case _:
+            params["prev_nav_url"] = url_for(INDEX_BP_DASHBOARD)
+
+    if request.method == "GET":
+        # Render the form on the initial GET request
+        return render_template("fund.html", **params)
```

Comment:
We can remove this altogether. `form.validate_on_submit()` already checks if it's a POST request. If it's not, then we will fall through to the end and do the render_template call there - which is identical. 🙏 

```suggestion
```